### PR TITLE
add o-neko and o-neko-catnip helm charts

### PR DESCRIPTION
**Pull request by philmtd**
I added our Helm charts for O-Neko and O-Neko Catnip here and added the Bitnami MongoDB chart as a dependency to the O-Neko chart.

### CODE REVIEW

File: charts/o-neko-catnip/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# O-Neko Catnip
+
+Helm Chart for [O-Neko Catnip](https://github.com/subshell/o-neko-catnip).
+
+## Requirements
+
+To install O-Neko Catnip with this chart you will need to create a secret manually which contains admin user
+credentials for O-Neko in fields named `username` and `password`. This secret needs to be referenced to in the
```
Comment (tom-schoener) [inline charts/o-neko-catnip/README.md:8]: remove the last `to`.

File: charts/o-neko/values.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  loadBalancerIP:
+  clusterIP:
+
+ingress:
+  # ingressClassName: nginx
+  hosts:
+  tls:
+  annotations: {}
+
+mongodb:
```
Comment (tom-schoener) [inline charts/o-neko/values.yaml:80]: I'd set `mongodb.enabled` to `true` explicitly.

Comment (tom-schoener) [reply charts/o-neko/values.yaml:80]: or to `false`. Whatever the better default is.

**Review by tom-schoener — CHANGES_REQUESTED** [review]

File: charts/o-neko-catnip/README.md (hunk 1 above)
Comment (philmtd) [reply charts/o-neko-catnip/README.md]: Done.

File: charts/o-neko/values.yaml (hunk 2 above)
Comment (philmtd) [reply charts/o-neko/values.yaml:80]: Done. It's now explicitly enabled.

**Review by tom-schoener — APPROVED** [review]

**Pull request merged**